### PR TITLE
Added Greeks

### PR DESCRIPTION
**Pull request by colin99d**
# Description

Gives users the ability to see Greeks for an option. Also gives us classes to do more with options in the future.

# How has this been tested?

Ran in the terminal. I will add tests before merging.

# Checklist:

- [x] Update [our Hugo documentation](https://gamestonkterminal.github.io/GamestonkTerminal/) following [these guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/website).
- [x] Update our tests following [these guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/tests).
- [x] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/GamestonkTerminal/GamestonkTerminal/blob/main/CONTRIBUTING.md).
- [x] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/scripts).

# Others
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My code passes all the checks pylint, flake8, black, ... To speed up development you should run `pre-commit install`.
- [x] New and existing unit tests pass locally with my changes. You can test this locally using `pytest tests/...`.

### CODE REVIEW

**Review by jmaslek — CHANGES_REQUESTED** [review]
Couple things.  

- I think second order greeks are overkill (at least displaying them.  Can keep the code)
- I was envisioning something along the lines of `chains` where these get shown in a table with all of the option chains

Also the way you handle expiration -- I don't think that is correct because 0dte options will diverge (ie gamma which is 1/T <> 1/0 ).  Probably a super subtle thing but I am curious how this typically gets handled.

File: gamestonk_terminal/stocks/options/op_helpers.py [hunk 1]
```diff
@@ -124,3 +128,149 @@ def rn_payoff(x: str, df: pd.DataFrame, put: bool, delta: int, rf: float) -> flo
...
+
+class Option:
+    def __init__(
+        self,
+        s: float,
+        k: float,
+        rf: float,
+        div_cont: float,
+        expiry: float,
+        vol: float,
```
Comment (jmaslek) [inline gamestonk_terminal/stocks/options/op_helpers.py:141]: Should be able to create a method that finds IV from the strike price

File: gamestonk_terminal/stocks/options/op_helpers.py [hunk 2]
```diff
@@ -124,3 +128,149 @@ def rn_payoff(x: str, df: pd.DataFrame, put: bool, delta: int, rf: float) -> flo
     "openInterest": {"format": "", "label": "Open Interest"},
     "impliedVolatility": {"format": "{x:.2f}", "label": "Implied Volatility"},
 }
+
+
+class Option:
+    def __init__(
+        self,
+        s: float,
+        k: float,
```
[the remark this answers is not in the document]
Comment (colin99d) [reply gamestonk_terminal/stocks/options/op_helpers.py:137]: S and K are standard in finance class, but I can rename for those outside.

File: gamestonk_terminal/stocks/options/op_helpers.py (hunk 1 above)
Comment (colin99d) [reply gamestonk_terminal/stocks/options/op_helpers.py:141]: This is handled in the view.

**Comment by jmaslek** [comment]
So like the `chains` command, the usual ones to display are:

Delta, Gamma (yes that's second order but its a big one (change in delta/increase in underlying), Theta, Vega and I would merge in the yfinance IV.

File: gamestonk_terminal/stocks/options/op_helpers.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jmaslek) [reply gamestonk_terminal/stocks/options/op_helpers.py:136]: yeah but someone gonna be looking at this without a finance background and ask WTF.  Also in general, I think the single letter variables are typically reserved for loops (something Didier has yelled at me for).

**Review by jmaslek — APPROVED** [review]

**Pull request merged**